CMIP6_experiment_id.json

```diff
@@ -3732,6 +3732,35 @@
             ],
             "tier":"1"
         },
+        "hist-GHG-cmip5":{
+            "activity_id":[
+                "DAMIP"
+            ],
+            "additional_allowed_model_components":[
+                "AER",
+                "CHEM",
+                "BGC"
+            ],
+            "description":"Historical well-mixed GHG-only run. Models with interactive chemistry schemes should either turn off the chemistry or use a preindustrial climatology of stratospheric and tropospheric ozone in their radiation schemes. This will ensure that ozone is fixed in all these simulations, and simulated responses in models with and without coupled chemistry are comparable. (CMIP5-era forcing)",
+            "end_year":"2020",
+            "experiment":"historical well-mixed GHG-only run (CMIP5-era forcing)",
+            "experiment_id":"hist-GHG-cmip5",
+            "min_number_yrs_per_sim":"163",
```

Comment:
Should this be 171 if the end_year=2020? In CMIP5, the end year was 2012 (and hence 163) but I prefer the current setting for end_year=2020 (171 yrs) which make them more comparable to CMIP6 runs.

Reply:
I don't see how the end year can be different from CMIP5.  We would need to add 8 years of CMIP5 GHG forcing to what's available.  Maybe I'm missing something.

Reply:
@taylor13 the extension beyond the historical forcings are RCP45, and so in theory the CMIP5 simulations could have been extended through to 2100 or beyond. For this reason, there is no inconsistency to extend the `*-cmip5` experiments from 2012 to 2020

Reply:
@slavakharin, good catch, caught that one (so it's now 171, 1850 to 2020 for all new `*-cmip5` experiments)

Reply:
@taylor13 I think your comment is also resolved, so will resolve this thread